data/breedsData.ts

```diff
@@ -4,7 +4,7 @@ export const breedsData = [
   "airedale",
   "akita",
   "appenzeller",
-  "australian-shepherd",
+  "australian/shepherd",
```

Comment:
Do not hardcode it like this. It's 100% dependent on API, so get list of possible breeds from API first.
1. So create an elegant hook like `useBreeds` and fetch/format everything inside.
2. You could also chek out awesome library to use instead of regular fetch which will take care of caching this for you: https://react-query.tanstack.com/